ledger/ledger-api-bench-tool/src/main/scala/com/daml/ledger/api/benchtool/services/TransactionService.scala

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.api.benchtool.services
+
+import com.daml.ledger.api.benchtool.metrics.LogOnlyObserver
+import com.daml.ledger.api.v1.ledger_offset.LedgerOffset
+import com.daml.ledger.api.v1.transaction_filter.{Filters, TransactionFilter}
+import com.daml.ledger.api.v1.transaction_service.{
+  GetTransactionTreesResponse,
+  GetTransactionsRequest,
+  GetTransactionsResponse,
+  TransactionServiceGrpc,
+}
+import io.grpc.Channel
+import org.slf4j.LoggerFactory
+
+final class TransactionService(channel: Channel, ledgerId: String) {
+  private val logger = LoggerFactory.getLogger(getClass)
+  private val service: TransactionServiceGrpc.TransactionServiceStub =
+    TransactionServiceGrpc.stub(channel)
+
+  // TODO: add filters
+  def transactions(party: String): LogOnlyObserver[GetTransactionsResponse] = {
+    val request = getTransactionsRequest(
```

Comment:
Maybe introduce a unique identifier for each stream open and then use it in log messages so that we know which one was started/completed/ended-in-error etc

Reply:
Yes, I'm going to add this when adding multi-stream configuration.